native_client/definitions.mk

```diff
@@ -63,9 +63,19 @@ PYTHON_PACKAGES := numpy${NUMPY_BUILD_VERSION}
 endif
 
 ifeq ($(TARGET),rpi3)
+PYVER := $(shell python -c "import platform; maj, min, _ = platform.python_version_tuple(); print(maj+'.'+min);")
```

Comment:
this will test the python version on the host platform, not on the target cross-compile, I dont see how this can correctly work